os-client/src/main/kotlin/io/provenance/scope/objectstore/client/CachedOsClient.kt

```diff
@@ -76,6 +94,13 @@ class CachedOsClient(val osClient: OsClient, osDecryptionWorkerThreads: Short, o
         )
     }
 
+    /**
+     * Fetch a jar from Object Store by hash
+     * @param [hash] the hash of the jar
+     * @param [encryptionKeyRef] the [PrivateKey][java.security.PrivateKey] to use to decrypt the jar
```

Comment:
There's a few places where the `PrivateKey` is mentioned alongside the `encryptionKeyRef` param, should we instead note that the key ref is rather used as a encryption/decryption provider?

Reply:
oh yeah, good catch, I'll go through those for consistency

Reply:
pushed that up